spec/lib/mpi/messages/find_profile_by_facility_spec.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'mpi/messages/find_profile_by_facility'
+
+describe MPI::Messages::FindProfileByFacility do
+  describe '.perform' do
+    subject do
+      described_class.new(facility_num:, dfn:).perform
+    end
+
+    let(:facility_num) { 'some-facility-num' }
+    let(:dfn) { 'some-dfn' }
+    let(:expected_identifier) { "#{facility_num}^PI^#{dfn}^#{MPI::Constants::VHA_FULL_IDENTIFIER}" }
+    let(:idm_path) { 'env:Envelope/env:Body/idm:PRPA_IN201305UV02' }
+    let(:parameter_list_path) { "#{idm_path}/controlActProcess/queryByParameter/parameterList" }
+
+    shared_examples 'successfully built request' do
```

Comment:
I don't see any specs that call this shared_example, prob should create a 'successful' context that calls this